packages/website/scripts/generate.js

```diff
@@ -0,0 +1,137 @@
+const path = require('path');
+const fs = require('fs/promises');
+const pick = require('just-pick');
+const rimraf = require('rimraf');
+const { getRepoRootPath, getKitDirs } = require('../../../scripts/kits-utils');
+
+const KIT_PAGE_RELATIVE_LAYOUT_PATH = '../../layouts/KitLayout.astro';
+
+function convertToFrontmatter(obj) {
+  return Object.entries(obj)
+    .map(([key, value]) => `${key}: ${obj[key]}`)
+    .join('\n');
+}
+
+function formatMarkdownFile(markdown, frontmatter) {
+  return `---
+# generated by scripts/kitgen at ${new Date().toISOString()}
+# edit KIT_PAGE_RELATIVE_LAYOUT_PATH in scripts/kitgen to change layout
+layout: ${KIT_PAGE_RELATIVE_LAYOUT_PATH}
+${frontmatter}
+---
+
+${markdown}`;
+}
+
+async function createDeleteFiles(kit, PagesPath) {
```

Comment:
Since `kit` is a boolean value here, I'd suggest changing this to `isKit`, making it more obvious that it's not being passed in a kit but a true/false on if it is a kit or not.